### PR TITLE
fix: Hot fix snapshot limit 8 strategies

### PR DESCRIPTION
**Pull request by chef-jojo**
super quick fix where snapshot api now limit to 8 strategies per request

### CODE REVIEW

File: src/views/Voting/helpers.ts [hunk 1]
```diff
@@ -181,34 +186,20 @@ function calculateVotingPower(scoresList: GetScoresResponse, voters: string[], s
...
 
     const total = cakeBalance
       .plus(cakeVaultBalance)
       .plus(cakePoolBalance)
       .plus(IFOPoolBalance)
-      .plus(cakeBnbLpBalance)
-      .plus(poolsBalance)
-      .div(TEN_POW_18)
-      .toFixed(18)
+      .plus(new BigNumber(cakeBnbLpBalance).times(TEN_POW_18))
```
Comment (memoyil) [inline src/views/Voting/helpers.ts:195]: I think we should keep the div and tofixed here

Comment (chef-jojo) [reply src/views/Voting/helpers.ts:195]: calculation on below
https://github.com/pancakeswap/pancake-frontend/blob/ce75a69b31bb369cc683141cfdbafbca474f237a/src/views/Voting/helpers.ts#L284-L294

Comment (memoyil) [reply src/views/Voting/helpers.ts:195]: 👍

**Comment by memoyil** [comment]
LGTM

**Comment by chefilip** [comment]
eslint ?

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**